src/components/avatar/avatar.html

```diff
@@ -1,59 +1,56 @@
-<article>
-    <h2 class="article-header">Avatars</h2>
```

Comment:
since `<article><h2>` part was repeated for each component, I move that part away